style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/sequelize/currency.md

## File overview

> There will be multiple files: a DB init script, your models, and your bot script. In the previous tutorial, we placed all of these in the same file. Having everything in one file is usually not how the real world does it, so we'll correct that.

Reviewer: The original sentence reads better imo. But I'd like to stray away from phrases like "the real world". Suggestions welcome.

Author: Maybe something pertaining to practices? Ex.

> ... usually not a good practice ...

Reviewer: ```suggestion
There will be multiple files: a DB init script, your models, and your bot script. In the previous tutorial, we placed all of these in the same file. Having everything in one file isn't an ideal practice, so we'll correct that.
```

};
```

> Like you see in the diagram above, our Users model will only have two attributes: a `user_id` primary key and a `balance`. A primary key is a particular attribute that becomes the default column used when joining tables together and is automatically unique and not `null`.

Reviewer: ```suggestion
Like you see in the diagram above, our Users model will only have two attributes: a `user_id` primary key and a `balance`. A primary key is a particular attribute that becomes the default column used when joining tables together, and is automatically unique and not `null`.
```

> Balance also sets `allowNull` to `false`, which means that both values have to be set in conjunction with creating a primary key; otherwise, the database would throw an error. This constraint guarantees correctness in our data storage. We never have null or empty values, ensuring that if we somehow forget to validate in the application that both values are not null, our database would do the final validation for us.

Reviewer: ```suggestion
Balance also sets `allowNull` to `false`, which means that both values have to be set in conjunction with creating a primary key; otherwise, the database would throw an error. This constraint guarantees correctness in our data storage. We'll never have null or empty values, ensuring that if we somehow forget to validate in the application that both values are not null, our database would do the final validation for us.
```

});
```

> We're defining an `.add()` method to our currency collection. We'll use it quite frequently, so having a method for it makes our lives easier. We'll also add a `.getBalance()` method so that we'll always get a number.

Reviewer: "It'll be used" is fine

Author: "be used" is passive, which we're trying to avoid.

storedBalances.forEach(b => currency.set(b.user_id, b));
```

> We want to sync our currency collection with the database for easy access later in our ready event.

Reviewer: I think I prefer the former. This can interpreted as "do this so that you can have easy access to it later _inside_ your ready event", instead of just "inside your ready event, do this so that you can have easy access to it later"

message.channel.send(`You've bought: ${item.name}.`);
```

> For users to search for an item without caring about the case, we use the `$iLike` modifier when looking for the name. Keep in mind that this may be slow if you have millions of items, so please don't put a million items in your shop.

Reviewer: ```suggestion
For users to be able to search for an item without caring about the letter casing, we use the `$iLike` modifier when looking for the name. Keep in mind that this may be slow if you have millions of items, so please don't put a million items in your shop.
```
